packages/siopv2-oid4vp-op-auth/src/agent/DidAuthSiopOpAuthenticator.ts

```diff
@@ -344,32 +357,62 @@ export class DidAuthSiopOpAuthenticator implements IAgentPlugin {
 
     const pex = new PEX()
     const verifiableCredentialsWithDefinition: Array<VerifiableCredentialsWithDefinition> = []
+    const dcqlCredentialsWithCredentials: Map<DcqlCredentialRepresentation, UniqueDigitalCredential> = new Map()
+
+    if (authorizationRequestData.presentationDefinitions !== undefined && authorizationRequestData.presentationDefinitions !== null) {
```

Comment:
if Array.isArray()

Reply:
Added if Array.isArray() check

Reply:
I didn't mean add. Just replace. Those "if  undefined and null" are not the way we check elsewhere anyway. Just do an if on the object. Array.isArray will never be true for null or undefined

Reply:
Using only Array.isArray() now